examples/read_parquet_cloud/src/main.rs

```diff
@@ -22,7 +22,14 @@ fn main() -> PolarsResult<()> {
         .select([
             // select all columns
             all(),
+            // and do some aggregations
+            cols(["fats_g", "sugars_g"]).sum().suffix("_summed"),
         ])
+        .with_predicate_pushdown(true)
```

Comment:
All those optimizations are default. We don't have to specify them.

Reply:
Removed, thanks for letting me know.